clients/hadoopfs/src/test/resources/contract/lakefs.xml

```diff
@@ -106,7 +106,10 @@
         <name>fs.contract.rename-overwrites-dest</name>
         <value>true</value>
     </property>
-
+    <property>
+        <name>fs.contract.create-visibility-delayed</name>
```

Comment:
Maybe worth a comment?  IIUC it says objects are not visible after `out = fs.create()`, only after `out.close()`.  Which makes sense, but is not clear.

Reply:
Done